exercises/practice/clock/.docs/instructions.md

```diff
@@ -5,3 +5,11 @@ Implement a clock that handles times without dates.
 You should be able to add and subtract minutes to it.
 
 Two clocks that represent the same time should be equal to each other.
+
+You will also need to impliment `.to_string()` for the `Clock` struct. We will be using this to display the Clock's data.  You can either do it via implimenting  `String::from` or using the [Display trait](https://doc.rust-lang.org/std/fmt/trait.Display.html).
+
+You might also want to add extra attributes like `#[derive(Debug, PartialEq)]` to the struct during testing.
+
+Enjoy!
```

Comment:
While this is a friendly close to the document, I'd prefer to keep the instructions document more matter-of-fact.